styledComponents/ScreenStyling.js

```diff
@@ -0,0 +1,95 @@
+import styled from "styled-components/native";
+
+export const ScreenContainer = styled.ImageBackground`
+  flex: 1;
+  align-items: center;
+  justify-content: space-around;
+  padding-top: 60px;
+`;
+
+export const Title = styled.Text`
+  color: #f4f4f4;
+  margin: 10% 0px 20px;
+  font-size: 30px;
+  text-align: center;
+  font-weight: bold;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  flex: 1;
+`;
+
+export const TextDescription = styled.Text`
+  letter-spacing: 3;
+  color: #f4f4f4;
+  text-align: center;
+  text-transform: uppercase;
+  font-size: 16px;
+  font-weight:bold;
+  margin: 0 10px;  
+  flex: 5;
+`;
+
+export const ScreenWrapper = styled.View`
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 100px;
+  flex: 5;
+`;
+
+export const ScreenButton = styled.TouchableOpacity`
+  letter-spacing: 3px;
+  alignItems: center;
+  backgroundColor: #f4f4f4;
+  padding: 20px;
+  margin: 0px 20px;
```

Comment:
We don't need to specify unit, if value is equal 0 - `margin: 0 20px;`

Reply:
Removed px from 0px.